learn/configuration/synonyms.md

```diff
@@ -2,8 +2,15 @@
 
 If multiple words have an equivalent meaning in your dataset, you can [create a list of synonyms](/reference/api/synonyms.md#update-synonyms). This will make your search results more relevant.
 
-In general, **a search on a word will return the same results as a search on any of its synonyms**.
-There is one exception to this rule, [detailed below](#multi-word-phrases).
+Words set as synonyms won't always return the same results. With the default settings, the `movies` dataset should return 547 results for `great` and 66 for `fantastic`. Let's set them as synonyms:
+
+<CodeSamples id="synonyms_guide_1" />
+
+With the new settings, searching for `great` returns 595 results and `fantastic` returns 423 results. This is due to various factors like [allowed typos](/learn/configuration/typo_tolerance.md#minwordsizefortypos) and [splitting the word](/learn/advanced/concat.md#split-queries) to find relevant documents. The search for `great` will allow only one typo and take into account all variations of `great`, e.g., `creatures`, `greatest`, etc. along with `fantastic`.
```

Comment:
A few comments on this:

1. I would remove "allowed" from "allowed typos"—it's not wrong, but I think it adds potential for confusion. Like, as a reader I immediately start wondering about the differences between allowed and disallowed typos;
2. "splitting the word" sounds really vague to me—words are a bit of a funny concept in Meilisearch, especially when taking languages like Japanese into account. I recommend "split queries" instead, which seems clearer and more technically correct;
3. On the last sentence we say "take into account all variations of `great`, e.g., `creatures`, `greatest`, etc"—is `creatures` supposed to be there? Perhaps "greater" was the intended word?

```suggestion
With the new settings, searching for `great` returns 595 results and `fantastic` returns 423 results. This is due to various factors like [allowed typos](/learn/configuration/typo_tolerance.md#minwordsizefortypos) and [splitting the word](/learn/advanced/concat.md#split-queries) to find relevant documents. The search for `great` will allow only one typo and take into account all variations of `great`, e.g., `creatures`, `greatest`, etc. along with `fantastic`.
```

Reply:
3. Yep, it's ` <em>creat</em>ures`. Should I use ` <em>creat</em>ures` instead of `creatures`? Would that make it clearer?

Reply:
Ah, yeah, that makes sense. What confused me is that we say:

a. "The search for `great` will allow only one typo"
b. "and take into account all variations of `great`, e.g., `creatures`, `greatest`, etc. along with `fantastic`."

So we have two cases, a and b, but only one example. I read the example as illustrating only the nearest situation, that is, variations of great.

GH marked this discussion as outdated, so I'll make a new comment with a suggestion where the examples are closer to each situation they're illustrating.